apps/docs/src/app/platform/component-docs/platform-object-attribute/platform-object-attribute-docs.module.ts

```diff
@@ -0,0 +1,41 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { ApiComponent } from '../../../documentation/core-helpers/api/api.component';
+import { API_FILES } from '../../api-files';
+import { SharedDocumentationPageModule } from '../../../documentation/shared-documentation-page.module';
+
+import { PlatformObjectAttributeHeaderComponent } from './platform-object-attribute-header/platform-object-attribute-header.component';
+import { PlatformObjectAttributeDocsComponent } from './platform-object-attribute-docs.component';
+import {
+    PlatformObjectAttributeExampleComponent,
+    PlatformObjectAttributeTruncateExampleComponent
+} from './platform-object-attribute-examples/platform-object-attribute-examples.component';
+import { PlatformObjectAttributeModule, PlatformLinkModule } from '@fundamental-ngx/platform';
```

Comment:
Suggestion:
Same as previous, maybe it will be better to split inner and outer imports?